dotcom-rendering/src/web/components/HeaderTopBarEditionDropdown.tsx

```diff
@@ -56,6 +58,15 @@ export const HeaderTopBarEditionDropdown = ({
 		},
 	];
 
+	if (isInEuropeTest)
+		links.push({
+			id: 'eur',
+			url: '/preference/edition/eur',
+			isActive: editionId === 'EUR',
+			title: 'Europe edition',
+			dataLinkName: 'nav3 : topbar : edition-picker: EUR',
+		});
+
```

Comment:
I would prefer if we did not mutate links, which can be achieved by spreading a array conditionally inside `links`.

```ts
const links = [
  "uk",
  "us",
  ...(isInEuropeTest ? ["eu"] : []),
] as const;
```

[TS Playground](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAlhAkmAogVwE4gA4FMAqu0MAvDALICGUAFgHQaVgAmIAtgBQCUMAfDAAY6AVgDcAKHGhIsADZwwAawikYAbXEwtMAERpFOgDSbteiEZNa61jgmToseQsQD86nbjQ6AujABc6t5cxr6UKtLQElLgECCyuHSyIADmHPJKEFxAA)